sandbox/lua/encoders/schema_influx_write.lua

```diff
@@ -0,0 +1,330 @@
+-- This Source Code Form is subject to the terms of the Mozilla Public
+-- License, v. 2.0. If a copy of the MPL was not distributed with this
+-- file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+--[=[
+Converts full Heka message contents to JSON for InfluxDB HTTP write API
+(new in InfluxDB v0.9.0).  Optionally includes all standard message fields
+as tags or fields and iterates through all of the dynamic fields to add as
+points (series), skipping any fields explicitly omitted using the `skip_fields`
+config option.  It can also map any Heka message fields as tags in the
+request sent to the InfluxDB API, using the `tag_fields` config option.
+Tags, timestamp/precision, database, and retention policy are defined once
+for all points submitted in the API call.  All dynamic fields in the Heka
+message are converted to separate series in the `points` array that is
+submitted to InfluxDB.
+
+Config:
+
+- name_prefix (string, optional, default "name")
+    String to use as the `name` key's prefix value in the generated JSON.
+    Supports interpolation of field values from the processed message, using
```

Comment:
You can just refer to the message field interpolation docs here, like we do in the [cbuf_librato docs](https://github.com/mozilla-services/heka/blob/dev/sandbox/lua/encoders/cbuf_librato.lua#L24).